src/App.tsx

```diff
@@ -152,11 +163,36 @@ class Board extends React.Component<BoardProps, BoardState> {
         this.setState(newState);
     }
 
-    public handlePress = (e: KeyboardEvent) => {
-        let i = parseInt(e.key, 10);
-        console.log(i);
+    /**
+     * deselect / unhighlight all highlighted squares
+     */
+    public deselectSquares(): void {
+        const newState = _.cloneDeep(this.state);
+        newState.highlighting = false;
+        newState.highlights = new Set<SquareAddress>();
+        this.setState(newState);
+    }
+
+    public handleGlobalKeyDown = (e: KeyboardEvent) => {
+        if (e.keyCode === 8) { // Delete Key
+            e.preventDefault();
+            this.deleteSelectedSquares();
+            return;
+        }
+        if (e.keyCode === 9) { // Tab Key
+            e.preventDefault();
+            this.toggleNumpadMode();
+            return;
+        }
+        if (e.keyCode === 91) { // Command Key
+            e.preventDefault();
+            const newState = _.cloneDeep(this.state);
+            newState.continueHighlighting = true;
+            this.setState(newState);
+            return;
+        }
+        const i = parseInt(e.key, 10);
```

Comment:
I would recommend doing this as:
1. Check if the keycode is between KEY_1 (49) and KEY_9 (57)
2. If it is, just subtract keycode by KEY_0 (48)

Reply:
You can learn more about the ascii encodings by typing `man ascii` in your terminal. Finally, I'd recommend referencing all ascii values by their hex notations (0x30) instead of the decimal notations. Hex is more common.